torch_xla/csrc/helpers.cpp

```diff
@@ -619,4 +619,18 @@ xla::StatusOr<xla::XlaComputation> XlaHelpers::WrapXlaComputation(
   return builder.Build(orig_result);
 }
 
+torch::lazy::Shape XlaHelpers::ConvertXlaShapeToLazy(const xla::Shape& shape) {
+  at::ScalarType scalar_type = TensorTypeToRawXlaType(shape.element_type());
```

Comment:
I guess it should be `TensorTypeFromXlaType`?

Reply:
Yep, my local dev is rebuilding tf so it was taking too long to rebuild and test lol